pkg/transform/snowplow_gtmss_preview.go

```diff
@@ -96,6 +115,10 @@ func extractHeaderValue(parsedEvent analytics.ParsedEvent, ctx, prop string) (*s
 			return nil, errors.New("invalid header value")
 		}
 
+		_, err = base64.StdEncoding.DecodeString(headerVal)
```

Comment:
I think we need to check for nil - only some events will have a header value attached, and when it's not attached, we want to proceed without setting a header, as opposed to failing it.

Reply:
We only do this decoding if there is some value: 

https://github.com/snowplow/snowbridge/blob/9c85766cfe18de50dcee8edb81079467d668ee14/pkg/transform/snowplow_gtmss_preview.go#L111

Reply:
Ah, silly me - you are right. Thanks for confirming!